components/raftstore-v2/src/raft/storage.rs

```diff
@@ -129,6 +132,18 @@ impl<EK: KvEngine, ER> Storage<EK, ER> {
     pub fn has_dirty_data(&self) -> bool {
         self.has_dirty_data
     }
+
+    #[inline]
+    pub fn set_flushed_epoch(&mut self, epoch: RegionEpoch) {
+        if util::is_epoch_stale(&self.flushed_epoch, &epoch) {
+            self.flushed_epoch = epoch;
+        }
```

Comment:
Better to clone epoch when we need.

```suggestion
    pub fn set_flushed_epoch(&mut self, epoch: &RegionEpoch) {
        if util::is_epoch_stale(&self.flushed_epoch, epoch) {
            self.flushed_epoch = epoch.clone();
        }
```

Reply:
good suggestion.